awx/ui/src/components/ResourceAccessList/DeleteRoleConfirmationModal.js

```diff
@@ -38,7 +37,7 @@ function DeleteRoleConfirmationModal({ role, username, onCancel, onConfirm }) {
         </Button>,
       ]}
     >
-      {isTeamRole() ? (
+      {isTeamRole() === 'Team' ? (
```

Comment:
I think we should rename this function since it's returning a string. The line being changed even seems like it could be have been a bug, since it was being used to conditionally render something even though the result is always a (truthy) string.